config/defaults.yml

```diff
@@ -82,6 +82,7 @@ common: &common
     - support
     - billing
   allow_registration: true
+  invite_required: true
```

Comment:
invite_required defaults to true since that's how it's implemented right now. Ideally it should be `false`, but we need more tests to make sure the app works seamlessly in both states.